.github/workflows/reuseablehcpkgrelease.yaml

```diff
@@ -0,0 +1,39 @@
+name: compare branches
+on:
+  workflow_call:
+   inputs:
+    main:
+     description:  "Main Branch"
+     required: true
+     type: string
+    releasing:
+      description: "releasing Branch"
+      required: true
+      type: string
+
+
+jobs:
+  compare:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: | 
+          if [ "$(git rev-parse ${{inputs.releasing}})" != "$(git rev-parse ${{inputs.main}})" ]; then 
+          echo "Branches are different launching package release" 
+          echo '::set-output name=diff::launch'
+          else 
+          echo "Same"
+          exit
+          fi
+        id: hcsh 
+  launch:
+    needs: compare
+    if: ${{needs.hcsh.outputs.diff}} == 'launch'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: | 
+          echo "Starting Release Process"
```

Comment:
lets complete it before PR, the script i'm currently using is in https://mana.mozilla.org/wiki/display/Hubs/Production+release+chore+chart

also I think it'd be nice to add a slack message at the end to notify the QAs, save one more manual step